scripts/sonic_sku_create.py

```diff
@@ -0,0 +1,776 @@
+#!/usr/bin/env python3
```

Comment:
This script should be Python 2 compatible and the shebang should read `#!/usr/bin/env python2` for 201911 branch.

Reply:
addressed in next commit.